lib/media/drm_engine.js

```diff
@@ -776,8 +776,8 @@ shaka.media.DrmEngine = class {
             config.persistentState = 'required';
           }
 
-          if (info.sessionTypes && info.sessionTypes.length) {
-            config.sessionTypes = info.sessionTypes;
+          if (info.sessionType && info.sessionType.length) {
```

Comment:
I think the length check is now unneeded.  If the length is zero, the string is already "falsey".